src/agecalc.rs

```diff
@@ -0,0 +1,45 @@
+use std::fs::File;
+use std::io::Write;
+use std::io::{BufReader, BufRead};
+
+struct Person{
+    fname : String,
```

Comment:
which naming convention is this? Use snake case convention for all variable names